source/common/res/features/import-notification/main.js

```diff
@@ -22,4 +23,3 @@
     setTimeout(poll, 250);
   }
 }());
-
```

Comment:
Yeah I use es-linter (I think that's the one but not home right now so can't check) in Atom which is why I was surprised to see that the line was gone.